baybe/searchspace/discrete.py

```diff
@@ -298,52 +308,113 @@ def from_simplex(
 
         # Validate non-negativity
         min_values = [min(p.values) for p in simplex_parameters]
+        max_values = [max(p.values) for p in simplex_parameters]
```

Comment:
The comment for this part 2 lines above says `# Validate non-negativity`, this should be adjusted.

Reply:
think it refers to what comes in the next line 312+